AntennaTracker/tracking.cpp

```diff
@@ -31,10 +31,14 @@ void Tracker::update_vehicle_pos_estimate()
  */
 void Tracker::update_tracker_position()
 {
+    struct Location temp_loc = current_loc;
+
     // update our position if we have at least a 2D fix
     // REVISIT: what if we lose lock during a mission and the antenna is moving?
-    if (!ahrs.get_position(current_loc) && (gps.status() >= AP_GPS::GPS_OK_FIX_2D)) {
+    if (gps.status() >= AP_GPS::GPS_OK_FIX_2D) {
```

Comment:
This doesn't look right.  Why are we trusting the GPS over the AHRS?

Reply:
I can put the if statements the other way round, the key thing is not to overwrite the current location when calling the ahrs get position

Reply:
if's swapped